library-best-practices/glossary.md

```diff
@@ -66,3 +81,55 @@ Java Dependency Glossary
 - **Linkage-matchable version** (said of a particular version of A in relation
   to all versions of B): There exists some version of B such that the version of
   A and the version of B are linkage-compatible.
+
+
+### Class usage graph
+
+- **Class usage graph**: a possibly cyclic directed graph of references between classes
+  in the classpath. The nodes (vertices) of the graph correspond to
+  Java classes and the directed edges are references between classes.
+  For example, when 'Class A' invokes method X on 'Class B',
+  the class usage graph holds an edge between the two nodes:
+
+  ```
+  [Class A] --(method X of class B)-> [Class B]
+  ```
+
+  In this case, 'Class A' is called the _source class_ of the reference and
```

Comment:
Should we call these "using class" and "referenced class"?

Reply:
"referenced class" sounds good. But "using class" doesn't for me.

How about "source class" and "referenced class"?

Reply:
Now it's 'source class' and 'target class' to follow terms in Guava Graph.